frontend/packages/client/src/components/Community/CommunityEditorProfile.js

```diff
@@ -143,13 +172,13 @@ function CommunityEditorProfile({
                 ? { border: '1px dashed #757575' }
```

Comment:
I think this one should be border: '2px dashed #757575' I know it's not related to this PR but just to keep borders consistency